src/partition/ProvidedBoundingBox.cpp

```diff
@@ -116,4 +125,15 @@ void ProvidedBoundingBox::computeBoundingBox()
   }
 }
 
+// these functions will be implemented in package 3
```

Comment:
Then, in principle, they should not be included here :)

Reply:
we need them to avoid having an abstract class, so we can instantiate from it for testing!

Reply:
This is nitpicking, but instead of the comment above, you could have added doxygen todos
`/// @todo Implement in next Package`